src/docs/product/releases/health/index.mdx

```diff
@@ -82,6 +82,21 @@ Each release is represented by a line in the chart, and you can navigate to the
 
 The chart does not display if you don't select a project or if you select multiple projects.
 
+
+### Adoption Stage
+
```

Comment:
```suggestion

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```